src/views/components/buttons.html

```diff
@@ -0,0 +1,34 @@
+<h1><u>Buttons</u></h1>
+<!-- Provides extra visual weight and identifies the primary action in a set of buttons -->
+<button type="button">Default Button</button>
+<!-- Deemphasize a button by making it look like a link while maintaining button behavior -->
+<button type="button" class="btn-light">Light Button</button>
+<button type="button" class="btn-link">Link Button</button>
+
+<div class="header">
+  <button type="button" class="btn-large">
+    <svg viewBox="0 0 100 100" class="icon icon-light">
+      <use xlink:href="#symbol-add"></use>
+    </svg>
+    <span>Add Booking</span>
+  </button>
+  <button type="button" class="btn-large">
+    <svg viewBox="0 0 100 100" class="icon icon-light">
+      <use xlink:href="#symbol-walkin"></use>
+    </svg>
+    <span>Add Walkin</span>
+  </button>
+  <button type="button" class="btn-large">Large Button</button>
+
+
+</div>
+
+<hr/>
+<pre>
+  &lt;button type="button"&gt;Default&lt;/button&gt;
```

Comment:
This is not right, don't worry about pre for now, we'll do it automatically later